test/test.js

```diff
@@ -102,7 +103,8 @@ describe('Puppeteer', function() {
       await neverResolves;
       expect(error.message).toContain('Protocol error');
     }));
-    it('userDataDir option', SX(async function() {
+    // Windows has issues running Chromium using a custom user data dir. It hangs when closing the browser.
```

Comment:
This seems to be the https://github.com/GoogleChrome/puppeteer/issues/918

Could you please refer to the issue in the comment so that we keep track of it?

Reply:
Done